src/status_im/contact/db.cljs

```diff
@@ -18,8 +18,8 @@
 (defn public-key->contact
   [contacts public-key]
   (when public-key
-    (get contacts public-key
-         (public-key->new-contact public-key))))
+    (or (get contacts public-key)
```

Comment:
just wondering how its different, why `not found` didn't work ?

Reply:
oh no, that's fine it worked before.

I changed this for performance reasons, as if you use:

`(get x y (some-func z))` 

`(some-func z)` is always executed (in our case is expensive as it makes a call to the bridge sync.

while if you use

`(or x (some-func z))` 
`(some-func z)` is only executed if `x` is falsy (i.e it short circuit)

Also sometime it's a bit unexpected behavior `(get ..)`, as `(get {:a nil} :a "default")` returns actually `nil`, which most of the times is not what you want (it does mean though that someone didn't clean up the map correctly)